cypress/e2e/tests/migration/applicationinventory/applications/filter.test.ts

```diff
@@ -118,16 +119,8 @@ describe(["@tier2"], "Application inventory filter validations", function () {
 
     it("Name filter validations", function () {
         Application.open();
-
-        var validSearchInput = applicationsList[0].name.substring(0, 15);
-        applySearchFilter(name, validSearchInput);
-        cy.wait(2000);
-        exists(applicationsList[0].name);
-        if (applicationsList[1].name.indexOf(validSearchInput) >= 0) {
-            exists(applicationsList[1].name);
-        }
-
-        clickByText(button, clearAllFilters);
+        //filtering substring of apps and validating application exists
```

Comment:
*nitpicl* the comment is redundant, the code is explanatory enough

Reply:
done